src/OpenTracing/Mock/MockSpan.cs

```diff
@@ -285,6 +292,11 @@ public LogEntry(DateTimeOffset timestamp, IDictionary<string, object> fields)
                 Timestamp = timestamp;
                 Fields = new ReadOnlyDictionary<string, object>(fields);
             }
+
+            public override string ToString()
+            {
+                return $"Timestamp: {Timestamp}, Fields: " + string.Join("; ", this.Fields.Select(e => $"{e.Key} = {e.Value}"));
```

Comment:
Just a note - unrelated change

Reply:
Agreed. Please remove this.

Reply:
Thanks for noticing, it just slipped into the PR from my local changes.